_data/chains/eip155-420420421.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Westend Asset Hub",
-  "chain": "WST",
+  "name": "Westend",
```

Comment:
Why is the `Asset Hub` suffix being stripped? Aren't all of the chains modified in this PR asset hubs?

Reply:
We made the decision that mentioning individual chains to the user is confusing. AssetHub is the designated place to interact with the whole of Polkadot (or its testnets). So when saying Polkadot you implicitly mean Polkadot AssetHub. We are trying to clean up the amount of words people need to learn. Polkadot itself doesn't have any user facing functionality. It will all be on AssetHub eventually.

Reply:
@FrederikBolding I confirm that this is the agreed on terminology.